sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -286,32 +225,39 @@ export interface SendBatchOptions {
 // @public
 export interface Subscription {
     close(): Promise<void>;
-    isRunning(): boolean;
+    isRunning: boolean;
 }
 
 // @public
 export interface SubscriptionEventHandlers {
     processClose?: ProcessCloseHandler;
     processError?: ProcessErrorHandler;
-    processEvents: ProcessEvents;
+    processEvent: ProcessEventHandler;
     processInitialize?: ProcessInitializeHandler;
 }
 
 // @public
-export interface SubscriptionOptions extends SubscriptionEventHandlers, EventProcessorOptions, EventProcessorBatchOptions {
+export interface SubscriptionOptions {
+    maxWaitTimeInSeconds?: number;
```

Comment:
Let's talk more about this offline. I'm really on the fence about it, primarily because the next possible iteration of this API will include having a "batch" of events and it won't seem so out of place there.

Reply:
Removed. It's really not worth the complication to the user.